sdk/tables/azure-data-tables/README.md

```diff
@@ -7,66 +7,56 @@ The Azure Data Tables client can be used to access Azure Storage or Cosmos accou
 [Source code][source_code] | [Package (PyPI)][Tables_pypi] | [API reference documentation][Tables_ref_docs] | [Samples][Tables_samples]
 
 ## Getting started
-The Azure Data Tables can be accessed using an Azure Storage or a CosmosDB account.
+The Azure Data Tables SDK can access an Azure Storage or CosmosDB account.
 
 ### Prerequisites
 * Python 2.7, or 3.5 or later is required to use this package.
 * You must have an [Azure subscription][azure_subscription] and either
-    * an [Azure storage account][azure_storage_account] or
+    * an [Azure Storage account][azure_storage_account] or
     * an [Azure Cosmos Account][azure_cosmos_account].
 
-#### Create a storage account
+#### Create account
 If you wish to create a new storage account, you can use [Azure Portal][azure_portal_create_account],
 [Azure PowerShell][azure_powershell_create_account], or [Azure CLI][azure_cli_create_account]:
 
-```bash
-# Create a new resource group to hold the storage account -
-# if using an existing resource group, skip this step
-az group create --name MyResourceGroup --location westus2
-# Create the storage account
-az storage account create -n MyStorageAccount -g MyResourceGroup
-```
-
-#### Creating a Cosmos DB
-If you wish to create a new cosmos storage account, you can use [Azure Cosmos DB][azure_create_cosmos].
-Create a Cosmos DB account `MyCosmosDBDatabaseAccount` in resource group `MyResourceGroup` in the subscription `MySubscription` and a table named `MyTableName` in the account.
-```bash
-az cosmosdb create --name MyCosmosDBDatabaseAccount --resource-group MyResourceGroup --subscription MySubscription
-az cosmosdb table create --name MyTableName --resource-group MyResourceGroup --acount-name MyCosmosDBDatabaseAccount
-```
-
+If you wish to create a new cosmos storage account, you can use [Azure Cosmos DB][azure_cli_create_cosmos].
```

Comment:
We should also link to docs on how to do this in the portal, same as storage above.